app/views/restrooms/_restroom.html.haml

```diff
@@ -1,4 +1,4 @@
-%div{:class => "listItem #{restroom.unisex? ? '' : 'not_unisex'} #{restroom.accessible? ? '' : 'not_accessible'}", "data-id" => restroom.id}
+%div{:class => "listItem #{restroom.unisex? ? '' : 'not_unisex'} #{restroom.accessible? ? '' : 'not_accessible'} #{restroom.changing_table? ? '' : 'no_changing_table'}", "data-id" => restroom.id}
```

Comment:
This could be done with a little bit of refactoring too.

```haml
- unisex_class = restroom.unisex? '' : 'not_unisex'
- accessible_class = restroom.accessible? '' : 'not_accessible'
- changing_table_class = restroom.accessible? '' : 'no_changing_table'

%div{ class: "listItem #{unisex_class} #{accessible_class} #{changing_table_class}",
          "data-id": restroom.id}
```